component/notification/src/main/java/org/exoplatform/social/notification/impl/SpaceNotificationImpl.java

```diff
@@ -114,9 +117,10 @@ public void spaceAccessEdited(SpaceLifeCycleEvent event) {}
   public void addInvitedUser(SpaceLifeCycleEvent event) {
     Space space = event.getSpace();
     String userId = event.getTarget();
-    
+    String SenderName = space.getEditor();
```

Comment:
Any manager could invite the user, I think the sender should be the one who did really send the invitation

Reply:
ok, i'll work on that